tool/metrics_monitor.py

```diff
@@ -0,0 +1,75 @@
+import time
+import datetime as dt
+import threading
+import sys
+import psutil
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+from core_data_modules.logging import Logger
+
+COLLECTION = 'pipeline_metrics'
+DEFAULT_INTERVAL = 3
```

Comment:
can you please add a comment here that this is in seconds?

Reply:
Added the comment. It's more descriptive now